releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 4.0.0-preview - July 23, 2020
+* Added `IConfigurationRefresher.SetDirty` method  to set the cached value for key-values registered for refresh as dirty. For users that use push notifications from event grid to receive configuration change updates, this allows the cached values to be revalidated on the next call to `RefreshAsync` or `TryRefreshAsync`.
+* Added support for deserializing configuration settings with JSON content type. [#191](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/191)
+* Added an interface `IConfigurationRefresherProvider` to provide access to instances of `IConfigurationRefresher` through dependency injection. This can be configured by adding a call to the following extension method on the service collection. [#167](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/167)
+   ```` csharp
+   public static IServiceCollection AddAzureAppConfiguration(this IServiceCollection services)
+   ````
+
```

Comment:
Looks like https://github.com/Azure/AppConfiguration-DotnetProvider/issues/166 is missed from the release notes. It's a breaking change. We should call it out and explain why.

Reply:
I will include this in the release notes for the AspNetCore package. We can skip it here since the breaking change in this issue wasn't part of this package before the current release.

Reply:
That's right.